api/src/pcapi/tasks/mails_tasks.py

```diff
@@ -0,0 +1,19 @@
+import pcapi.core.mails.transactional as transactional_mails
+from pcapi.tasks.decorator import task
+from pcapi.tasks.serialization.mails_tasks import WithdrawalChangedMailRequest
+
+
+@task("withdrawal_details_changed_notification", "/mails/withdrawal_notif")
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion: wording** ‏On gagne pas grand chose à utiliser la contraction ici
```suggestion
@task("withdrawal_details_changed_notification", "/mails/withdrawal_notification")
```

Reply:
c'est pas faux